setup.md

```diff
@@ -40,10 +40,10 @@ GUAC components work together]({{ site.baseurl }}{%link guac-components.md %}).
 - [Docker](https://docs.docker.com/get-docker/)
 - [Docker Compose](https://docs.docker.com/compose/install/)
 - [Git](https://git-scm.com/downloads)
-- [Go](https://go.dev/doc/install)
+- [Go](https://go.dev/doc/install) (v1.20.1)
```

Comment:
hmm, actually I don't think it needs to be so specific. What error were you getting @noah-de?

Reply:
I am currently on v1.20.3 and do not get any errors.

Reply:
The machine that I was installing on had a much older version of Go, resulting in many errors.

I don't recall the specific errors and am on a different system today (no browser history).

@pxp928  I just changed it to what you had requested in https://github.com/guacsec/guac-docs/pull/78#discussion_r1266589384